src/pages/popup/Popup.tsx

```diff
@@ -9,6 +9,15 @@ import withErrorBoundary from '@src/shared/hoc/withErrorBoundary';
 const Popup = () => {
   const theme = useStorage(exampleThemeStorage);
 
+  const injectContentScript = async () => {
+    const [tab] = await chrome.tabs.query({ currentWindow: true, active: true });
+    console.log(tab.id)
+    chrome.scripting.executeScript({
```

Comment:
This results in

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'executeScript')
```

I think you need to have the `scripting` permission in the manifest.  You also need to have the `activeTab` permission, or have the content script allowed as described here:

https://developer.chrome.com/docs/extensions/reference/api/scripting